katzenmint/cmd/katzenmint/register.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+
+	katzenmint "github.com/hashcloak/Meson/katzenmint"
+	"github.com/katzenpost/core/crypto/eddsa"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/privval"
+	"github.com/tendermint/tendermint/rpc/client/http"
+)
+
+var (
+	registerValidatorCmd = &cobra.Command{
```

Comment:
By the way, since we expose the registration command to users, do we still need to keep the field Config.Membership now?

Reply:
I was thinking that too. I'll remove Membership config lol.